pythcat/repwithna.py

```diff
@@ -1,12 +1,22 @@
-def repwithna(df):
+import pandas as pd
+import numpy as np
+
+def repwithna(df, rmvpunc=False, format=None):
     """
     Replace uninformative strings (eg. empty strings like '') in the data frame with NAs, 
```

Comment:
Non-crucial: might suggest replacing the words 'NA' with 'NaN' just to be consistent with python and your example